projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -200,4 +204,69 @@ export function updateVDKMenu(commands: CommandRegistry) {
       }
     }
   });
+
+  commands.addCommand('jp-vdk:menu-create-deployment', {
```

Comment:
Some general feedback on this file: There's a lot of duplicated code that can be factored out, especially related to error handling.

Reply:
yes, I was thinking about this problem, as well. Maybe I will be moving adding the component into the menu in the component file and to factor out some duplicate code.